x/msg_authorization/internal/types/router.go

```diff
@@ -0,0 +1,53 @@
+package types
```

Comment:
I don't think we want to duplicate `Router` here, rather we just want to re-use baseapp's `Router` definition.

Reply:
Sure, will make the changes accordingly